statsd/statsd.go

```diff
@@ -83,12 +83,28 @@ traffic instead of UDP.
 */
 const UnixAddressPrefix = "unix://"
 
-// Client-side entity ID injection for container tagging
+/*
+Client-side entity ID injection for container tagging.
+*/
 const (
 	entityIDEnvName = "DD_ENTITY_ID"
 	entityIDTagName = "dd.internal.entity_id"
 )
```

Comment:
I don't think we need to keep those variables around, we could hardcode them in the `ddEnvTagsMapping` like the other ones.